github/workflows/ci.yml

```diff
@@ -0,0 +1,51 @@
+name: CI
+
+on:
+  push:
+    branches: [ main, develop ]
+  pull_request:
+    branches: [ main, develop ]
+
+jobs:
+  backend:
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: ./backend
+
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Use Node.js
+      uses: actions/setup-node@v2
+      with:
+        node-version: '14'
```

Comment:
<div>

<strong>Bito Code Review Agent Run</strong> <a href=https://github.com/omkardongre/Food-Ordering-App/pull/3#issuecomment-2352179665>#f974b3</a><span> - 09/16/2024, 07:22 am</span>
<br>

<div><small>🔴</small> High importance</div>

<div id="suggestion">
<div id="issue">
<b>Issue:</b> The specified Node.js version (14) is outdated and no longer supported, which may lead to security vulnerabilities and compatibility issues.
</div>
<div id="fix">
<b>Fix:</b> Update the Node.js version to a currently supported LTS version, such as 18.x or 20.x.
</div>
<details>
<summary>
<b>Code suggestion</b>
</summary>
<div id="code">

```
 @@ -19,7 +19,7 @@
      - name: Use Node.js
        uses: actions/setup-node@v2
        with:
 -        node-version: '14'
 +        node-version: '18'
 @@ -39,7 +39,7 @@
      - name: Use Node.js
        uses: actions/setup-node@v2
        with:
 -        node-version: '14'
 +        node-version: '18'
```
</div>
</details>
</div>
<br>


</div>

Is this a valid issue, or was it incorrectly flagged by the Agent?
- [ ] it was incorrectly flagged